opm/simulators/flow/Transmissibility_impl.hpp

```diff
@@ -344,12 +344,12 @@ update(bool global, const TransUpdateQuantities update_quantities,
             auto computeHalfMean = [&inside, &outside, &computeHalf, &ntg, &harmonicMean]
                                    (const auto& halfComputer, const auto& prop)
             {
-                auto half = computeHalf(halfComputer, prop[inside.elemIdx], prop[outside.elemIdx]);
```

Comment:
If we're changing this anyway, maybe we could rename the result to `onesideded` or similar.  "Half" really isn't a good name for this.

Reply:
ok.